custom-elements/reactions/HTMLHyperlinkElementUtils.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<title>Custom Elements: CEReactions on HTMLHyperlinkElementUtils interface</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<meta name="assert" content="href, protocol, username, password, host, hostname,
+  port, pathname, search, hash of HTMLHyperlinkElementUtils interface must have
+  CEReactions">
```

Comment:
Not clear what these assert metas add; better to remove them if possible.

Reply:
OK, I will remove them.